components/Footer/styles.css

```diff
@@ -1,192 +1,228 @@
 /** @define Footer; use strict */
 
+.button-primary {
+  border: none;
+  outline: none;
+  display: inline-block;
+  width: 100%;
+  font-size: 14px;
+  font-weight: 900;
+  color: #FFFFFF;
+  letter-spacing: 1.5px;
+  max-width: 300px;
+  height: 44px;
+  line-height: 43px;
+  text-align: center;
+  position: relative;
+  text-transform: uppercase;
+  background: #F8AA31;
+  -webkit-border-radius: 3px;
+  -moz-border-radius: 3px;
+  border-radius: 3px;
+  transition: all 200ms ease-in;
+  cursor: pointer; }
+
+.button-primary:hover {
+  background-color: #f7a018;
+  color: #FFFFFF;
+}
+
 .Footer {
+  margin: 8px;
```

Comment:
remove this margin, it's doubling up with the margin around all the content already